app/assets/stylesheets/partials/_event_box.scss

```diff
@@ -0,0 +1,51 @@
+.event-box {
+    width: 350px;
+    padding: 2.1rem 1.5rem;
+
+    border: 1px solid $v3-color_green-light;
+
+    * { color: $color_font-light !important; }
```

Comment:
Wondering if there is a way to do this without using this !important rule. Maybe target the text element classes individually? I feel like we might need to refactor some of this out when we combine this component with the rest of the updated front page anyways so it could be ok to let this sit for now.

Reply:
Yeah, I could increase the specificity